macos/Classes/MobileScannerPlugin.swift

```diff
@@ -26,6 +26,8 @@ public class MobileScannerPlugin: NSObject, FlutterPlugin, FlutterStreamHandler,
     var scanWindow: CGRect?
     
     var detectionSpeed: DetectionSpeed = DetectionSpeed.noDuplicates
+
+    var timeoutSeconds: Double = 0
```

Comment:
How would you find a little enhancement, introducing `TimeInterval` instead of raw Double? As a _typealias_ it doesn't change anything in the below logics but is more obvious that we are operating on seconds. 
```suggestion
    var timeout: TimeInterval = 0
```